user_class/user.rb

```diff
@@ -1,2 +1,108 @@
 class User
+  attr_reader :all
+
+  def initialize
+    @all = []
+    @VALID_ATR = %i(first_name last_name email age address)
```

Comment:
This is not an instance variable. You should define it as a constant in the class since the fields are not going to change.